lib/taskana-core/src/test/java/acceptance/classification/QueryClassificationAccTest.java

```diff
@@ -39,30 +40,30 @@ public QueryClassificationAccTest() {
     public void testQueryClassificationValuesForColumnName() {
         ClassificationService classificationService = taskanaEngine.getClassificationService();
         List<String> columnValueList = classificationService.createClassificationQuery()
-            .listValues("NAME", null);
+            .listValues(ClassificationQueryColumnName.NAME, null);
```

Comment:
a static import could be nice for these use cases..

Reply:
I have added the static imports now.

Reply:
I just noticed I have forgotten to add it in the AccessItemQuery, I'm at it now.